packages/components/src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./Spinner.css";
+
+export interface SpinnerProps {
+  readonly small?: boolean;
```

Comment:
What do you think of changing this to…

```suggestion
  readonly size?: "small" | "base";
```

Then below we can set size to be `"base"` by default.

Reply:
Good callout, this is much better. Changed! 😄